src/eh1/spi.rs

```diff
@@ -8,8 +8,7 @@
 //! ## Usage
 //!
 //! ```
-//! # use eh1 as embedded_hal;
-//! use embedded_hal::spi::{SpiBus, SpiBusWrite};
+//! use eh1::spi::SpiBus;
```

Comment:
Since these docs are for public users (which would import `embedded_hal`), I'd prefer to use the `embedded_hal` alias in here (and in all other rustdocs below as well).